filebeat/docs/autodiscover-hints.asciidoc

```diff
@@ -63,7 +63,7 @@ a list of configurations.
 
 ["source","yaml",subs="attributes"]
 -------------------------------------------------------------------------------------
-co.elastic.logs/raw: "[{\"containers\":{\"ids\":[\"${data.container.id}\"]},\"multiline\":{\"negate\":\"true\",\"pattern\":\"^test\"},\"type\":\"docker\"}]"
+co.elastic.logs/raw: "[{"containers\":{"ids\":[\"${data.container.id}"]},\"multiline\":{"negate\":\"true\",\"pattern\":\"^test\"},\"type\":\"docker\"}]"
```

Comment:
@nik9000 I'm not sure about this change. The changes I made here got the output to match the original, but looking at this a second time, I'm not sure what the original author intended. Thoughts?

Reply:
It looks to me like the author meant the `\` to be literally in the example. This is another place where I'd drop `subs="attributes"`.

Reply:
Oh doh. You're right. I didn't notice that the author included that attribute. I will fix.